extensions/kusto/src/objectExplorerNodeProvider/hdfsProvider.ts

```diff
@@ -0,0 +1,336 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as azdata from 'azdata';
+import * as vscode from 'vscode';
+import * as fspath from 'path';
+import * as fs from 'fs';
+import * as nls from 'vscode-nls';
+const localize = nls.loadMessageBundle();
+
+import * as Constants from '../constants';
+import { IFileSource, IFile, File } from './fileSources';
+import { CancelableStream } from './cancelableStream';
+import { TreeNode } from './treeNodes';
+import * as utils from '../utils';
+import { IFileNode } from './types';
+
+export interface ITreeChangeHandler {
+	notifyNodeChanged(node: TreeNode): void;
+}
+export class TreeDataContext {
+
+	constructor(public extensionContext: vscode.ExtensionContext, public changeHandler: ITreeChangeHandler) {
+
+	}
+}
+
+export abstract class HdfsFileSourceNode extends TreeNode {
```

Comment:
does kusto extension support HDFS?

Reply:
No it does not. This is part of some cleanup we were planning to done once we checkin to main. I'll look into it in this PR.